Integrate-CircleCI.md

```diff
@@ -0,0 +1,75 @@
+# CircleCI Integration
+This guide will demonstrate how to configure CircleCI and run your test script with Kobiton on CircleCI.
+> Note: The configuration file used in the example is based on CircleCI 2.0 configuration file format.
+
+## Table of contents
+[Prerequisites](#prerequisites)
+
+[1. Configuring GitHub repository to be used with CircleCI](#1.-configuring-github-repository-to-be-used-with-circleci)
+
+[2. Integrating CircleCI with configured GitHub repository](#2.-integrating-circleci-with-configured-github-repository)
```

Comment:
Table of contents
+ [Prerequisites](#prerequisites)
+ [Configuring GitHub repository to be used with CircleCI](#1.-configuring-github-repository-to-be-used-with-circleci)
+ [Integrating CircleCI with configured GitHub repository](#2.-integrating-circleci-with-configured-github-repository)